docs/bootc-images.md

```diff
@@ -55,4 +55,79 @@ as part of your `RUN` invocations.   This will perform early detection
 of some incompatibilities but is not a strict requirement today and will not be
 in the future.
 
+# Building and using `bootc` compatible images in the real world
 
+As of November 2023, there are two primary ways of generating a 
+`bootc` compatible base image and both require the use of `rpm-ostree`.
+
+The easiest and most straight-forward way is to use `rpm-ostree compose image`
+in order to produce a `bootc` compatible OCI image from a [treefile](https://coreos.github.io/rpm-ostree/treefile/)
+or manifest file. This is the most direct way to be able to produce an OCI
+image that can be pushed to a registry or used by other container tools.
+
+The alternative method is to produce and `ostree` commit
+(i.e. using `rpm-ostree compose commit`) and then encapsulating the `ostree`
+commit in an OCI image via `rpm-ostree compose container-encapsulate`. This
+may be preferable if you already have a build process that produces `ostree`
+commits as an output (e.g. using osbuild to produce `ostree` commit artifacts.)
+
+The requirement for both methods is that your treefile/manifest **MUST** include
+the `bootc` package in list of packages included in your compose.
+
+## Building a custom `bootc` compatible image
```

Comment:
I'm OK with this but isn't it somewhat covered already by linking to e.g. https://github.com/coreos/layering-examples ?

(I think it'd be best to centralize that and clean it up alongside centos-boot )